content/en/docs/ops/configuration/mesh/configuration-scoping/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Configuration Scoping
+description: Shows how to scope configuration in Istio, for operational and performance benefits.
+weight: 10
```

Comment:
If you want this doc to be first in the section then you should make the weight <10, e.g., 5. If you want it last it should be >50. Otherwise you need to pick a number to place it where you want, relative to the other doc weights.
```suggestion
weight: 60
```